doc/advanced-topics.rst

```diff
@@ -110,15 +110,22 @@ to the definition of a build context.
 - the name of an alternative toolchain
 
 Note that at the moment, there is no official support for
-cross-compilation in OCaml. Jbuilder supports the two following
-opam-cross-x repositories:
-
-- `opam-cross-windows <https://github.com/whitequark/opam-cross-windows>`_
-- `opam-cross-android <https://github.com/whitequark/opam-cross-android>`_
-
-To build Windows binaries using opam-cross-windows, write ``windows``
-in the list of targets. To build Android binaries using
-opam-cross-android, write ``android`` in the list of targets.
+cross-compilation in OCaml. Jbuilder supports the opam-cross-x
+repositories from the `ocaml-cross organization on github
+<https://github.com/ocaml-cross/>`, such as:
+
+- `opam-cross-windows <https://github.com/ocaml-cross/opam-cross-windows>`_
+- `opam-cross-android <https://github.com/ocaml-cross/opam-cross-android>`_
+- `opam-cross-ios <https://github.com/ocaml-cross/opam-cross-ios>`_
+
+In particular:
+
+- to build Windows binaries using opam-cross-windows, write ``windows``
+  in the list of targets
+- to build Android binaries using opam-cross-android, write
+  ``android`` in the list of targets
+- to build Android binaries using opam-cross-ios, write ``ios`` in the
```

Comment:
should be "to build IOS binaries ..." on the last one

Reply:
Indeed, thanks. I fixed it in 9fbfc272ebe5066ab8ca9509b53427bee4559600